include/caf/events/module_state_event.h

```diff
@@ -90,12 +113,13 @@ static inline bool module_flags_check_zero(const struct module_flags *mf)
 }
 
 /**
- * @brief Test single module bit
+ * @brief Test single module bit.
+ *
+ * @param[in] mf         Pointer to module flags variable.
+ * @param[in] module_idx The index of the module whose bit should be tested.
```

Comment:
Avoid `should` in docs of any kind. It's ambiguous.

Reply:
ok, changed